ui/src/shared/components/TimeSeries.tsx

```diff
@@ -126,13 +128,21 @@ class TimeSeries extends Component<Props, State> {
   private pendingCheckStatuses: CancelBox<StatusRow[][]> = null
 
   public componentDidMount() {
+    const {cellID, setCellMount} = this.props
     this.observer = new IntersectionObserver(entries => {
       entries.forEach(entry => {
         const {isIntersecting} = entry
-        if (!this.isIntersecting && isIntersecting && this.pendingReload) {
+        const reload =
```

Comment:
```suggestion
        const shouldReload =
```

for clarity here, since there's a function called `reload`, and it's easy to mix up a verb representing a function and a noun representing a variable

Reply:
I used to feel the same exact way.  until typescript.  now what used to be "what is this thing" become very clear.

Reply:
@121watts i don't follow - there's no type hinting in this code, there's no context saying what this is, beyond the code itself. how does typescript help a reader understand the intent of the code here, or communicate what these things are?